ceph-iscsi.spec

```diff
@@ -48,6 +48,7 @@ Requires:       python-rtslib >= 2.1.fb68
 Requires:       python-cryptography
 Requires:       python-flask >= 0.10.1
 Requires:       python-configshell >= 1.1.fb25
+Requires:       python2-requests
```

Comment:
Nit: is this the name of the package under SUSE? Google implies it's just `python-requests`

Reply:
I'm not sure now, but I think it's python2-requests. I had looked it up here:

https://software.opensuse.org/package/python2-requests
but I do see:
https://software.opensuse.org/package/python-requests

For both, it does not show a SLES rpm. I just see the open suse one.

For opensuse, when I did a zypper install python2-requests it installed that rpm.

I just tried "zypper install python-requests" and the rpm that gets installed is python2-requests.

Adding @ricardoasmarques  just in case.

Reply:
Dang, but I did mess up the RHEL 7 rpm. It is named python-requests :)